p503/arith_amd64_test.go

```diff
@@ -21,21 +21,26 @@ const (
 	kUse_MULXandADxX = 1 << 2
 )
 
+func resetCpuFeatures() {
+	HasBMI2 = cpu.X86.HasBMI2
+	HasADXandBMI2 = cpu.X86.HasBMI2 && cpu.X86.HasADX
```

Comment:
It's still a shame that we need so many copies of these values...

Reply:
It is. I don't understand why we can't refer from ASM to variables in different package. But because of that we need this dumb copy.